### PR TITLE
Release/0.1.0

### PR DESCRIPTION
**Pull request by andre-urbani**
### What

Skeleton code

### How to review

Confirm additions are correct

### Who can review

Anyone

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -11,7 +11,7 @@ all: audit test build
 
 .PHONY: audit
 audit:
-	go list -m all | nancy sleuth
+	go list -json -m all | nancy sleuth --exclude-vulnerability-file ./.nancy-ignore
```
Comment (redhug1) [inline Makefile:14]: Why has "--exclude-vulnerability-file ./.nancy-ignore" been added ?
I thought the latest plan was to not have that anymore (as its not needed / wanted) ?
Its now not in to many other projects.
Please check with Neill.

File: config/config.go [hunk 2]
```diff
@@ -24,10 +32,17 @@ func Get() (*Config, error) {
...
-		BindAddr:                   "localhost:26200",
+		BindAddr:                   "localhost:26300",
 		GracefulShutdownTimeout:    5 * time.Second,
 		HealthCheckInterval:        30 * time.Second,
 		HealthCheckCriticalTimeout: 90 * time.Second,
+		KafkaAddr:                  []string{"localhost:9092"},
+		KafkaVersion:               "1.0.2",
+		KafkaOffsetOldest:          true,
+		KafkaNumWorkers:            1,
+		HelloCalledGroup:           "dp-import-cantabular-dimension-options",
```
Comment (redhug1) [inline config/config.go:43]: Can you have a look to see what other apps that use Kafka have renamed their 'HelloCalledGroup' and similar to something more appropriate ?
As this seems not quite right to me.

**Review by redhug1 — CHANGES_REQUESTED** [review]
Please run:
go mod tidy
and push the changes.
(and if the other comments make sens to change ...)
Thank you

File: config/config.go (hunk 2 above)
Comment (ChrysmOre) [reply config/config.go:43]: For the very first PR we left the boilerplate as is with the last service, the update for the proper names being in the first skeleton update

Comment (redhug1) [reply config/config.go:43]: OK

File: Makefile (hunk 1 above)
Comment (DavidSubiros) [reply Makefile:14]: My fault. It was from my review. I didn't know we were not supposed to have that anymore.

**Review by redhug1 — APPROVED** [review]

**Pull request merged**